src/System.Management.Automation/engine/Modules/RemoveModuleCommand.cs

```diff
@@ -84,6 +84,11 @@ protected override void ProcessRecord()
 
             if (FullyQualifiedName != null)
             {
+                // TODO:
+                // Paths in the module name may fail here because
+                // they the wrong directory separator or are relative.
+                // Fix with the code below:
+                // FullyQualifiedName = FullyQualifiedName.Select(ms => ms.WithNormalizedName(Context, SessionState.Path.CurrentLocation.Path)).ToArray();
```

Comment:
What's the concern about putting this fix in place? Regression? Perf degradation?

Reply:
Just keeping the fix constrained in this PR. It shouldn't cause regressions or perf degradations, but to ensure there are no regressions (since this PR is a regression fix itself) I just left it as a comment.

Reply:
Will you open issue to track all those `todo` comments? They can be easily forgotten :)

Reply:
Did you open an issue to track those todo's?

Reply:
Next on my list